src/desktop/apps/article/components/__tests__/Article.test.js

```diff
@@ -46,7 +46,7 @@ describe('<Article />', () => {
       published_at: '2017-05-19T13:09:18.567Z',
       contributing_authors: [{ name: 'Kana' }],
     })
-    const rendered = mount(<ArticleLayout article={article} templates={{}} />)
+    const rendered = shallow(<ArticleLayout article={article} templates={{}} />)
     rendered.find(InfiniteScrollArticle).length.should.equal(1)
```

Comment:
Shallow to avoid calling `componentDidMount` in text component -- assuming it should be safe here because we are only checking that components exist, and not calling instance methods in sub-components.